engine/src/main/java/org/terasology/rendering/dag/DefaultStateChangeStorage.java

```diff
@@ -15,20 +15,19 @@
  */
 package org.terasology.rendering.dag;
 
-import java.util.List;
+import java.util.HashMap;
+import static org.lwjgl.opengl.GL11.GL_FILL;
 
 /**
- *
+ * TODO: Add javadocs
  */
-public class RenderPipelineExecutor {
-
-    private List<PipelineTask> taskList;
+class DefaultStateChangeStorage extends HashMap<Class<? extends StateChange>, StateChange> {
```

Comment:
Oh, I didn't notice the `static` property in the definition. I'll look into this in more detail.